src/Server/Resource/DoctrineResource.php

```diff
@@ -348,6 +348,11 @@ public function create($data)
 
         $this->getObjectManager()->flush();
 
+        $results = $this->triggerDoctrineEvent(DoctrineResourceEvent::EVENT_CREATE_FLUSH, $entity, $data);
+        if ($results->last() instanceof ApiProblem) {
```

Comment:
This is confusing: if an API has already flushed then the data is persisted.  If we return an ApiProblem after persistence the client will not expect the data to have been persisted.  Perhaps _FLUSH will be unable to return an ApiProblem?